phpBB/phpbb/cache/service.php

```diff
@@ -18,6 +18,12 @@
 */
 class service
 {
+	/** @var string Name of event used for cache purging */
+	private const CACHE_PURGE_EVENT = 'core.garbage_collection';
```

Comment:
I think the naming of this constant is a bit confusing because this is not the cache purge event, but an (existing) event that will now also trigger the deferred purge. What do you think of: `PURGE_DEFERRED_ON_EVENT`?

Reply:
Changed as requested.